string name
string description
---
bool success
string message